test/router.js

```diff
@@ -0,0 +1,231 @@
+const test = require('tap').test
+const Fastify = require('fastify')
+const fastifyWebsocket = require('../')
+const websocket = require('websocket-stream')
+const request = require('request')
```

Comment:
I would prefer if we do not use the `request` client module. It has been deprecated by the maintainers, and it does not have a good track recently lately with semver-major changes.